### ISSUE TITLE
"Add new step here" appears on top of the steps after closing the curtain

### ISSUE THREAD

**Issue by fominykh** [WEKIT-ECS/MIRAGE-XR#1376]
related to #1374 
![Screenshot_20230501-191029_MirageXR](https://user-images.githubusercontent.com/10297277/235494753-5c1175bc-76f2-4ca4-93a0-1e2eeb87f4db.jpg)

The "add new step here" should be positioned below the current step. The list of steps should be scrolled to the position of the current step.

If the user is created step after step, this button should always be in the bottom, under the steps.
